wiki/compatibility.md

```diff
@@ -0,0 +1,8 @@
+## Compatibility
+
+* Maven 3.6.1
+* JDK 8
```

Comment:
similar here.

If we are not sure of the compatibility, we can mention that we have tested with this compatibility versions of jdk and maven

Reply:
Yes. Mentioned as we tested with these mentioned versions of maven and JDK.